VideoLocker/src/main/java/org/edx/mobile/util/AppUpdateUtils.java

```diff
@@ -9,49 +9,52 @@
 import android.view.View;
 import android.widget.Toast;
 
-import org.edx.mobile.BuildConfig;
+import com.google.inject.Inject;
+
 import org.edx.mobile.R;
 
 /**
- * Utility class for interacting with an app store, or the Play Store specifically.
+ * Utility class for updating the app.
  */
-public final class AppStoreUtils {
+public final class AppUpdateUtils {
     // Make this class non-instantiable
-    private AppStoreUtils() {
+    private AppUpdateUtils() {
         throw new UnsupportedOperationException();
     }
 
+    @Inject
+    private static Config config;
+
     /**
      * @param context A Context to query the applications info.
      *
-     * @return Whether there is any app store or web browser registered on the device.
+     * @return Whether there are any apps registered to handle the update URIs.
      */
-    public static boolean hasAppStore(@NonNull final Context context) {
+    public static boolean canUpdate(@NonNull final Context context) {
         final PackageManager packageManager = context.getPackageManager();
         final Intent intent = new Intent(Intent.ACTION_VIEW);
         intent.addCategory(Intent.CATEGORY_BROWSABLE);
-        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        final Config.AppUpdateUrisConfig appUpdateUrisConfig = config.getAppUpdateUrisConfig();
+        intent.setData(Uri.parse(appUpdateUrisConfig.getNativeUri()));
         if (intent.resolveActivity(packageManager) != null) return true;
-        intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
-                BuildConfig.APPLICATION_ID));
+        intent.setData(Uri.parse(appUpdateUrisConfig.getWebUri()));
         return intent.resolveActivity(packageManager) != null;
     }
 
     /**
-     * Open an app store to display the app, or a web browser displaying it's web page if no app
-     * store is available.
+     * Open an native app or website on a web browser to update the app.
```

Comment:
You're commenting on an older version of this javadoc, but this error was still retained in the latest version. Now fixed in commit e53fdcf9a32adaf821326a1a217570b9bbb79df7.